.eslintrc.json

```
@@ -0,0 +1,21 @@
+{
+  "extends": ["bloq", "prettier"],
```

Thinking if we should add `bloq/node` here too.

```suggestion
  "extends": ["bloq", "bloq/node", "prettier"],
```

---

Added here [dc618db](https://github.com/hemilabs/crypto-shortener/pull/4/commits/dc618dbf92c817c1f09582114c028dc5070180c5) However, do note that I had to disable `node/no-missing-import` because as the tests are in `js`, the eslint rule fails to detect the importing of ts code. We are covered by `knip` on this, so it is not a big deal (I tested it and knip detects an incorrect import)